README.md

```diff
@@ -30,6 +30,15 @@ Finally, run the image:
 $ docker run rust-test-app
 ```
 
+Environment variables
+---------------------
+
+Application developers can use the following environment variables to configure the runtime behavior of this image:
+
+NAME        | Description
+------------|-------------
+PROXY       | Use a cargo proxy during assembly
```

Comment:
Should this be `HTTP_PROXY` which seems like the standard way to use an HTTP proxy?

Reply:
I think `PROXY` is better, as cargo's proxy setting is [libcurl format](https://curl.haxx.se/libcurl/c/CURLOPT_PROXY.html) and allows users to write it with schema not only `proxy=https://...` or `proxy=https://...`, but also `proxy=socks5h://...`.

So, in the same way, `PROXY=socks5h://...` is comfortable for the s2i-rust users,

Reply:
sgtm